shared/actions/chat.js

```diff
@@ -312,13 +360,24 @@ function * _loadMoreMessages (): SagaGenerator<any, any> {
 
   const yourName = yield select(usernameSelector)
   const messages = (thread && thread.thread && thread.thread.messages || []).map((message, idx) => _unboxedToMessage(message, idx, yourName, conversationIDKey)).reverse()
+  let newMessages = []
+  messages.forEach((message, idx) => {
```

Comment:
Don't need to make this change but you can use reduce for cases like this instead of mutating newMessages